paasta_tools/prune_completed_pods.py

```diff
@@ -40,6 +40,14 @@ def parse_args():
         required=False,
         type=int,
     )
+    parser.add_argument(
+        "-p",
+        "--pending-minutes",
+        help="Minutes since the pod was scheduled. Terminates pods whose phase is Pending based on time since scheduled. "
+        "Including pod status Pending/ContainerCreating/Terminating.",
```

Comment:
TIL that pods shown as `Terminating` in kubectl are actually in the Pending phase!

Reply:
I also got confused between pod statuses and phases haha